fastly/tls_custom_certificate_test.go

```diff
@@ -201,3 +201,21 @@ func TestClient_UpdateCustomTLSCertificate_validation(t *testing.T) {
 		t.Errorf("bad error: %s", err)
 	}
 }
+
+func TestClient_CreateCustomTLSCertificateInput_default_values(t *testing.T) {
+	t.Parallel()
+
+	certificateInput := CreateCustomTLSCertificateInput{}
```

Comment:
I'm not quite sure what these tests are for; they are creating an instance of a struct which contains a `bool` field, which will always be false as there is no method in Go to have a 'default value' for a struct field.

Reply:
im a golang novice and felt better about the default behavior with a test.

if the test is redundant / unnecessary then im happy to remove

Reply:
Ahh, no worries then... please remove them as they are only testing to ensure that Golang's standard behavior hasn't changed :-)

Reply:
totally